safe-hts-precompile/SafeHTS.sol

```diff
@@ -435,7 +435,7 @@ library SafeHTS {
         (bool success, bytes memory result) = precompileAddress.call(
             abi.encodeWithSelector(IHederaTokenService.isToken.selector, token));
         (responseCode, isToken) = success ? abi.decode(result, (int32, bool)) : (HederaResponseCodes.UNKNOWN, false);
-        require(responseCode == HederaResponseCodes.SUCCESS, "Safe query for if valid token at address failed!");
+        require(responseCode == HederaResponseCodes.SUCCESS, "Safe is token failed!");
```

Comment:
This reads weird.  While all the other variants split out the words I think we should consider keeping this camelCase - `Safe isToken failed!` or `Safe 'is token' failed`

Reply:
Updated.